operators/operator_sdk/osdk-multi-arch-support.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="osdk-multiple-platform-support"]
+= Configuring Operator projects to support multiple architectures and operating systems
+include::_attributes/common-attributes.adoc[]
+:context: osdk-multi-arch
+
+toc::[]
+
+Operator projects that support multiple architectures and operating systems, or _platforms_, can run on more Kubernetes and {product-title} clusters than Operator projects that support only a single platform. Example architectures include `amd64`, `arm64`, `ppc64le`, and `s390x`. Example operating systems include Linux, IBM Z, and Windows.
```

Comment:
While IBM's z/OS is an operating system, I don't think you can build a container for it. I would probably keep the OS list to Linux and Windows. We sometimes see MacOS for client binaries.

Reply:
Addressed in https://github.com/openshift/openshift-docs/pull/64355/commits/cdcc6e4df48bcafea8897b9038d5d83e07a0dec7

Reply:
Addressed in https://github.com/openshift/openshift-docs/pull/64355/commits/a2a3c6dae140f265de1729bb7b4022f56b0dcd07